plugins/deployment_manager/Migration.ts

```diff
@@ -5,7 +5,7 @@ export interface Actions<T> {
   prepare: (dm: DeploymentManager, govDm: DeploymentManager) => Promise<T>;
   enact: (dm: DeploymentManager, govDm: DeploymentManager, t: T) => Promise<void>;
   enacted?: (dm: DeploymentManager, govDm: DeploymentManager) => Promise<boolean>;
-  verify?: (dm: DeploymentManager, govDm: DeploymentManager) => Promise<void>;
+  verify?: (dm: DeploymentManager, govDm: DeploymentManager, preMigrationBlockNumber: number, postMigrationBlockNumber: number) => Promise<void>;
```

Comment:
Open to other suggestions as well. We could also have `enact` output a new type and add as an argument in `verify`.

Reply:
Oh interesting 🤔